tests/unittests/lib.collections.art

```diff
@@ -702,10 +702,21 @@ do [
 
     debug split [[1 2 3] [4 5 6] [7 8]]
 
-    ;path
-    ; temporarily disable
-    ; see issue: #872
-    ; debug split.path "directory/wofilerld"
+
+    debug split.path "directory\\wofilerld"
+    debug split.path "D:\\User\\user\\.arturo"
+    debug split.path "/usr/bin"
+
+    p1: "directory\\wofilerld"
+    p2: "D:\\User\\user\\.arturo"
+    p3: "/usr/bin"
+
+    debug split.path p1
+    debug split.path p2
+    debug split.path p3
+    split.path 'p1, debug p1
+    split.path 'p2, debug p2
+    split.path 'p3, debug p3
```

Comment:
```suggestion
    debug split.path p1
    debug split.path p2
    split.path 'p1, debug p1
    split.path 'p2, debug p2
```

Since I removed the `p3` from above, let's remove this one too...